src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+
+    public Ball getRandomBall() {
+        int randomNumber = random.nextInt(0, 100);
+        return new Ball(colorSupplier.getRandomColor(), randomNumber);
```

Comment:
extract random.nextInt(100) to the method and inline in function calling:
```suggestion
        return new Ball(colorSupplier.getRandomColor(), getRandomColor());
```